helm/presets/local.yaml

```diff
@@ -49,13 +66,13 @@ presto:
       annotations:
         nginx.ingress.kubernetes.io/use-regex: "true"
       hosts: [storage.hindsight.local/minio.*]
-global:
-  objectStore:
-    bucketName: hindsight-object-storage
-    region: local
-    accessKey: hindsightAccessKey
-    accessSecret: hindsightAccessSecret
-    minio:
-      enabled: true
-      port: 9000
-      scheme: "http://"
+
+vernemq:
+  enabled: true
+  service:
+    type: LoadBalancer
```

Comment:
Why does verne need to be a load balancer but. acquire does not?  I think you've explained this to me once before but I forgot lol

Reply:
Since Acquire is over HTTP we can keep that an internal-only ClusterIP and have an ingress object route the traffic from the Nginx LoadBalancer service to the Acquire ClusterIP service. Since MQTT and MQTTS are not over HTTP, they'd need to be LoadBalancer type-services to be reachable outside of the cluster